src/js/util.js

```diff
@@ -23,12 +23,26 @@ const showStep = (step, stepList) => {
     return
   }
 
+  const stepperNode = closest(step, Selectors.STEPPER)
+
   const activeStep = stepList.filter(step => step.classList.contains(ClassName.ACTIVE))
   if (activeStep.length) {
     activeStep[0].classList.remove(ClassName.ACTIVE)
   }
+  for (let stepItem in stepList) {
```

Comment:
can you use, `forEach` here please, I don't see any interest using `for ... in` here, because we have a regular array, not a key value object

Reply:
I don't understand how that's possible, because we already use forEach on the `stepList` here: https://github.com/Johann-S/bs-stepper/blob/ea8a982db46ec86eda428d95589f6094cad01eaf/src/js/index.js#L17